main_test.go

```diff
@@ -544,6 +605,24 @@ func TestGetVMName(t *testing.T) {
 	}
 }
 
+func TestCheckIPv6Capable(t *testing.T) {
+	server := httptest.NewServer(nil)
+	defer server.Close()
+	overrideHTTP(server)
+	want := true
+	http.HandleFunc("http://metadata.google.internal/computeMetadata/v1/instance/network-interfaces/0/ipv6s",
+		func(w http.ResponseWriter, r *http.Request) {
+			if r.Header.Get("Metadata-Flavor") != "Google" {
+				http.Error(w, "Missing Metadata-Flavor", 403)
+				return
+			}
+			w.Write([]byte("6970:7636:2061:6464:7265:7373:2062:6162"))
+		})
+	if got := isIPv6Capable(); got != want {
```

Comment:
Should we have a test case where `isIPv6Capable()` returns false?

I see that other tests don't do that. I don't remember why. But, if we *can* test both cases here, that would be good I think.